src/components/Signup/tests/Signup.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import Signup from "../../Signup";
+ 
+describe("Signup component", () => {
+  test("pages renders correctly", () => {
+    const {queryByLabelText, queryByRole} = render(<Signup/>);
+    expect(queryByLabelText("Email")).toBeTruthy();
+    expect(queryByLabelText("Password")).toBeTruthy();
+    expect(queryByRole("button")).toBeVisible();
+    //TODO: check if button is rendered.
+  });
```

Comment:
If you're just testing if elements are rendered into the DOM, you can use the same methods from `screen`. This avoids having to keep using destructuring queries from `render(...)`. Kent Dodds talks about it [here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen).

Since you're using `jest-dom`, you could change `.toBeTruthy()` to be `toBeInTheDocument()`, since it's a little more accurate in stating that those elements are rendered. The same for the button as well.

Reply:
:+1 @shelbyhurns